xapian-applications/omega/query.cc

```diff
@@ -1547,7 +1563,6 @@ eval(const string &fmt, const vector<string> &param)
 			ket += '>';
 		    }
 		}
-
```

Comment:
The combined result of adding and removing code here is an arbitrary whitespace change, completely unrelated to the patch, which is a bad thing.

Otherwise, looking good now.